assets/js/main.js

```diff
@@ -162,8 +168,45 @@ $(() => {
 
     $("#toggle-search-all").on("change", function() {
         $("[data-type='nearby']").toggleClass("is-hidden");
+        $("[data-type='baseUnit']").toggleClass("is-hidden");
         $("[data-type='all']").toggleClass("is-hidden");
         $("[data-output='location']").toggleClass("is-hidden");
         updateResults(clubs, coords, this.checked);
     });
-});
\ No newline at end of file
+    $("#toggle-unit").on("change", function() {
+        var el = $("[for='toggle-unit']");
+        var currentUnit = el.html();
+        if(currentUnit == "Imperial"){
```

Comment:
Could you follow the convention? There are spaces between the closing parentheses and opening braces, and newlines after closing braces.